ERASABLE_ASSIGNMENTS.s

```diff
@@ -546,8 +546,8 @@ TRM03FLG	=	026D		# REQUEST TO		NO REQUEST TO
 TRM03BIT	=	BIT4
 
 # BIT 3 FLAG 1
-SLOPESW		= 	027D		# ITERATE WITH BIAS	ITERATE WITH REGULA
-					# METHOD IN ITERATOR	FALSI METHOD IN 
+SLOPESW		= 	027D		# ITERATE WITH BIAS	ITERATE WITH REGULAR
```

Comment:
`REGULA` is what is used in the [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0049.jpg)